submissions/collabcloud.md

```diff
@@ -0,0 +1,48 @@
+# Hackathon Submission: CollabCloud
+
+## GitHub handles of Team Members  
+[dfordp](https://github.com/dfordp)
+
+
+## Project Title
+_What is the name of your project?_
+
+CollabCloud
+
+## Project Description    
+
+This platform is catered to video creators and media houses allowing all the necessary features required to succeed and the ever-evolving  landscape of content creation.
+
+Key features include :
+
+- Handling multiple Social Media accounts from one single application 
+- Preprocessing content before upload to ensure high quality across different platforms , in line with it's unqiue internal video transcoding.
+- Easy Division of labour and communication regarding the content , allowing for effcient communication.
+- Securing social media profiles from potential attackers and hackers
+- All the tools a creator could need for their content in a single place , eg Social Media Engagement Inducing Tags, Finetuned models for analyzing and creating content, Scheduling Social media posts.
+
+
+
+## Inspiration behind the Project  
+
+I have had the pleasure of working as video editor and social media strategist for some really cool creators and agencies during my intial college years. I felt like there were certain things that could immensely impacted my productivity for the better, so this application is kind of like a gift to myself and challenge that I am taking to enhance my development skills
+
+## Tech Stack    
+
+- Next.js
+- Appwrite Authenthication
+- PostgresSQL
+- AWS Elastic Media Transcoder
+- Youtuber , Instagram API
+- Tailwind CSS
+- Accernity UI
+- Vercel
+- Cloudflare R2
+
+## Project Repo  
+
+https://github.com/Fada-Club/collabcloud
+
+## Demo Video/Photos  
+
+(under development will add as soon it's complete)
```

Comment:
@dfordp please ensure the demo is added here today as we are at the last day of the hackathon